awx/ui_next/src/api/models/JobTemplates.js

```diff
@@ -24,8 +23,11 @@ class JobTemplates extends InstanceGroupsMixin(NotificationsMixin(Base)) {
     return this.http.get(`${this.baseUrl}${id}/launch/`);
   }
 
-  associateLabel(id, label) {
-    return this.http.post(`${this.baseUrl}${id}/labels/`, label);
+  associateLabel(id, label, orgId) {
+    return this.http.post(`${this.baseUrl}${id}/labels/`, {
+      name: label.name,
+      organization: orgId,
```

Comment:
this call is used to generate a brand new label and to associate and existing label.